.github/workflows/python-ci.yml

```diff
@@ -50,5 +49,7 @@ jobs:
       run: |
         pip install -e .[dev]
         pip install -U tox-gh-actions
+    - name: Start MinIO instance
+      run: docker-compose -f python/docker-compose.ci.yml up -d
     - working-directory: ./python
-      run: tox
+      run: tox -- -m "marked or not marked"
```

Comment:
I don't think we want the tests marked `s3` to run by default locally, so I added `-m "not s3"` in the tox.ini file. Here what I'm doing is overriding that setting for the CI.

I couldn't find a `-m` argument dedicated for directing pytest to run **all** tests, including unmarked tests, so I do a bit of a trick here: although there is no group of tests named `marked`, telling pytest to run "marked or not marked" results in all tests running.